spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryGirl.define do
+
+  factory :user do |u|
+    sequence(:email) { |n| "example#{n}@yahoo.com" }
+    u.password { 'password' }
+    u.password_confirmation { 'password' }
+  end
+
+  factory :vault do |v|
+    v.name { "Vault" }
+    v.exposure { rand(0..2) }
+    v.user_id { rand(1..10) }
+  end
+
+  factory :snippet do |s|
+    s.name { 'Snippet' }
+    s.language { rand(0..30) }
+    s.code { 'Test test test test' }
+    s.vault_id { rand(1..100) }
```

Comment:
Yeah a slack team would be good! 

Sent from my iPhone

> On 05 Sep 2016, at 10:23, Salvatore Gentile <notifications@github.com> wrote:
> 
> In spec/factories.rb:
> 
> > +    sequence(:email) { |n| "example#{n}@yahoo.com" }
> > +    u.password { 'password' }
> > +    u.password_confirmation { 'password' }
> > +  end
> > +
> > +  factory :vault do |v|
> > +    v.name { "Vault" }
> > +    v.exposure { rand(0..2) }
> > +    v.user_id { rand(1..10) }
> > +  end
> > +
> > +  factory :snippet do |s|
> > +    s.name { 'Snippet' }
> > +    s.language { rand(0..30) }
> > +    s.code { 'Test test test test' }
> > +    s.vault_id { rand(1..100) }
> Before start working on new features I want to setup a community on Slack or something similar to enhance communication between contributors. I will also create a website to promote and explain my idea. Thank you for all your help!
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>